ddtrace/contrib/pylons/__init__.py

```diff
@@ -22,6 +22,22 @@
 Global Configuration
 ~~~~~~~~~~~~~~~~~~~~
 
+.. py:data:: ddtrace.config.pylons['distributed_tracing']
+
+   Whether to parse distributed tracing headers from requests received by your pylons app.
+
+   Can also be enabled with the ``DD_BOTOCORE_DISTRIBUTED_TRACING`` environment variable.
```

Comment:
```suggestion
   Can also be enabled with the ``DD_PYLONS_DISTRIBUTED_TRACING`` environment variable.
```

Reply:
🤦

Reply:
I also used `override_config` in the test case to set the setting to False, but I can just directly reference the `config` object directly. Is there any difference between the two?